services/grpc/grpc.js

```diff
@@ -160,6 +162,9 @@ class Grpc extends EventEmitter {
     Lightning.removeAllListeners('subscribeInvoices.data')
     Lightning.removeAllListeners('subscribeTransactions.data')
     Lightning.removeAllListeners('subscribeChannelGraph.data')
+    Lightning.removeAllListeners('subscribeChannelGraph.error')
```

Comment:
No listeners were added for `subscribeChannelGraph.error`

Reply:
updated. added forwarding of data/error events for all subscriptions